nemo/collections/nlp/data/token_classification/punctuation_capitalization_dataset.py

```diff
@@ -142,6 +142,17 @@ class PunctuationCapitalizationDataConfigBase:
     tar_shuffle_n: int = 1
     """The size of shuffle buffer of `webdataset`. The number of batches which are permuted."""
 
+    shard_strategy: Optional[str] = 'scatter'
```

Comment:
This one is a great finding! We also need same description in https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/punctuation_and_capitalization.rst in table "Parameters for tarred dataset".